### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chinkysight**
I have updated the `README.md` solving this kind of confusion https://github.com/dart-lang/mockito/issues/448

### CODE REVIEW

**Comment by chinkysight** [comment]
@srawlins please review my pull request

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ The [`when`], [`thenReturn`], [`thenAnswer`], and [`thenThrow`] APIs provide a
 stubbing mechanism to override this behavior. Once stubbed, the method will
 always return stubbed value regardless of how many times it is called. If a
 method invocation matches multiple stubs, the one which was declared last will
-be used.
+be used. It is worth noting that stubbing only works on methods of a mocked class e.g `MockCat` in this case.
```
Comment (srawlins) [inline README.md:110]: This actually applies to stubbing _and_ to verifying.

You could write "Stubbing and verifying only work..." or you could even just say "These APIs only work..."

WDYT?

Comment (chinkysight) [reply README.md:110]: It's great, I will update it 😊

Comment (chinkysight) [reply README.md]: @srawlins I have updated the docs, please have a look

**Review by srawlins — APPROVED** [review]

**Comment by srawlins** [comment]
Thanks!

**Pull request merged**